app/.gitignore

```diff
@@ -1 +1,10 @@
+*.iml
```

Comment:
Good job

Reply:
Is that something I shouldn't have on gitignore??

Reply:
I meant Good job.... lol... I think "good job" is a good thing right??  Good job in the entire file

Reply:
lol